src/mediarithmics/plugins/common/BasePlugin.ts

```diff
@@ -533,8 +533,12 @@ export abstract class BasePlugin {
 
     this.app.use(bodyParser.json({type: "*/*", limit: "5mb"}));
     
-    this.logger = winston.createLogger();
-    this.logger.add(new winston.transports.Console());
+    this.logger = winston.createLogger({
```

Comment:
So, apparently we broke the ability to log JSON without having to do explicit JSON.stringify (which is quite CPU intensive) when we upgraded winston to the 3.x release.

We should add the format.splat() to fix that.

See:
```
const winston = require("winston");

const object = {hello: "world"};

const brokenLog = winston.createLogger({
    format: winston.format.simple(),
    transports: [
        new winston.transports.Console()
    ]
});

brokenLog.info("[broken] Simple test %j", object);

const log = winston.createLogger({
    format: winston.format.combine(
        winston.format.splat(),
        winston.format.simple()
    ),
    transports: [
        new winston.transports.Console()
    ]
});

log.info("Simple test %j", object);
```

returns:
```
info: [broken] Simple test %j
info: Simple test {"hello":"world"}
```